content/media-player/next_steps/_index.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Next steps"
+weight = 6
+chapter = true
+pre = "5. "
++++
+
+# Next Steps
+
+{{<mermaid>}}
+flowchart LR
+    id1(Upload)-->id2(Model)-->id3(Visualize)-->id4(Track)-->id5(Next steps)
+    style id1 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id2 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id3 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id4 fill:#f5f5f5,stroke:#333,stroke-width:1px
+    style id5 fill:#f5f5f5,stroke:#6638B8,stroke-width:3px
+{{</mermaid >}}
+
+
```

Comment:
It feels like the Next step is not logical /necessary here as in the previous one we were already recommending to swap visualisations to their own data.

Reply:
Yeah I get what you mean, I'm thinking of adding a final step to link back to the visualisation chapter after the modelling your own data next step to show how to swap the tableau data sources.